packages/suite/src/views/dashboard/components/PortfolioCard/index.tsx

```diff
@@ -27,6 +27,11 @@ const Body = styled.div`
     flex: 1;
 `;
 
+const StyledDropdown = styled(Dropdown)`
+    display: flex;
+    height: 38px;
```

Comment:
this is an issue of the `DashboardSection` component, I checked 🙏

Reply:
Okey, can you fix it in some of your next PRs then please